From 2711475ad6b7ac23df75759afd89c9948b23a61b Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 13 Jan 2021 09:09:16 +0800
Subject: [PATCH] input: touchscreen: tp_suspend: Remove set but unused
 variable 'ret'

drivers/input/touchscreen/tp_suspend.h:77:6: warning: variable 'ret' set but not used [-Wunused-but-set-variable]

Fixes: ef0b86754ecd ("driver: input: touchscreen: gt9xx: add ebc suspend notify support")
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: I1fd66d6b8664e6f09cc4aad1544bfef424565dc0
---
 drivers/input/touchscreen/tp_suspend.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/input/touchscreen/tp_suspend.h b/drivers/input/touchscreen/tp_suspend.h
index afa978fec3ef..96f63cb2fc00 100644
--- a/drivers/input/touchscreen/tp_suspend.h
+++ b/drivers/input/touchscreen/tp_suspend.h
@@ -74,14 +74,13 @@ static int ebc_notifier_callback(struct notifier_block *self,
 		unsigned long action, void *data)
 {
 	struct tp_device *tp;
-	int ret;
 
 	tp = container_of(self, struct tp_device, ebc_notif);
 
 	mutex_lock(&tp->ops_lock);
 
 	if (action == EBC_FB_BLANK)
-		ret = tp->tp_suspend(tp);
+		tp->tp_suspend(tp);
 	else if (action == EBC_FB_UNBLANK)
 		tp->tp_resume(tp);
 
-- 
2.35.3

